web/packages/teleterm/src/mainProcess/runtimeSettings.ts

```diff
@@ -73,6 +73,7 @@ function getRuntimeSettings(): RuntimeSettings {
       `--addr=${tshAddress}`,
       `--certs-dir=${getCertsDir()}`,
       `--prehog-addr=${staticConfig.prehogAddress}`,
+      `--kubeconfigs-dir=${getKubeConfigsDir()}`,
```

Comment:
Let's assign the result of `getKubeConfigsDir` to a variable and then use the variable instead of calling `getKubeConfigsDir` twice in `getRuntimeSettings`.